ban/core/models.py

```diff
@@ -117,6 +117,7 @@ def housenumbers(self):
 
 
 class HouseNumber(Model):
+    CEA_FORMAT = '[\dAB]{2}\d{3}[234679ABCEGHILMNPRSTUVXYZ]{5}'
```

Comment:
@christopheprudent are you able to explain a bit why those numbers and letters and not others? Just for readability of the code :)